.github/workflows/check-urls.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       - name: Checkout  # for having the sources of the local action
         uses: actions/checkout@v4
         # download and unzip the ab-results-*.json generated by job-matrix: check-urls
-      - uses: actions/download-artifact@v4
+      - uses: actions/download-artifact@v3
```

Comment:
so this fixes it?

Reply:
It looks like it. I tried some other versions suggested in the issue like `v4.1.8` and those did not work. I'm still a little unclear on the underlying issue, but I think this could be a fine workaround while we keep an eye on the issue for an upstream fix. I will open a new PR with this change only.

Reply:
https://github.com/EbookFoundation/free-programming-books/pull/11230